package/MDAnalysis/lib/distances.py

```diff
@@ -1242,6 +1243,7 @@ def transform_StoR(coords, box, backend="serial"):
 
 @check_coords('coords1', 'coords2')
 def calc_bonds(coords1, coords2, box=None, result=None, backend="serial"):
+    start = time.perf_counter()
```

Comment:
This isn't how we do benchmarks in MDAnalysis, please see @hmacdope's comment here about how we use ASV for benchmarking https://github.com/MDAnalysis/mdanalysis/issues/3205#issuecomment-1079613028

Reply:
@IAlibay i'll do it according to https://github.com/MDAnalysis/mdanalysis/issues/3205#issuecomment-1079613028

Reply:
@IAlibay After reading https://github.com/MDAnalysis/mdanalysis/issues/3205#issuecomment-1079613028 and related pages i did not understand that where to add banchmarks? ,which files i have to edit ? what should i write?
please help me this.

> This isn't how we do benchmarks in MDAnalysis, please see @hmacdope's comment here about how we use ASV for benchmarking [#3205 (comment)](https://github.com/MDAnalysis/mdanalysis/issues/3205#issuecomment-1079613028)

Reply:
@IAlibay Now I have added banchmarks according to https://github.com/MDAnalysis/mdanalysis/issues/3205#issuecomment-1079613028 .please review it again